scripts/zkai_attention_block_statement_bridge_gate.py

```
@@ -0,0 +1,610 @@
+    )
+    return output.getvalue()
+
+
+def require_output_path(path: pathlib.Path | None, suffix: str) -> pathlib.Path | None:
+    if path is None:
+        return None
+    candidate = pathlib.Path(os.path.abspath(path if path.is_absolute() else ROOT / path))
+    try:
+        candidate.relative_to(EVIDENCE_DIR.resolve())
+    except ValueError as err:
+        raise AttentionBlockStatementBridgeError(f"output path must stay in docs/engineering/evidence: {path}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)